tests/unit/logicblocks/event/store/adapters/test_postgres.py

```diff
@@ -117,5 +130,150 @@ async def mock_close(_):
         assert not closed
 
 
+def normalize_whitespace(text: str) -> str:
+    return " ".join(text.split())
+
+
+def event(fixed_timestamp, name="event", value="value"):
+    return NewEvent(
+        name=name,
+        payload={"data": value},
+        observed_at=fixed_timestamp,
+        occurred_at=fixed_timestamp,
+    )
```

Comment:
There's `NewEventBuilder` and `StoredEventBuilder` which might be useful either in implementing these helper functions or instead of these helper functions.